test/unit/test-mysqlDao.js

```diff
@@ -73,7 +89,18 @@ const userDao = new UserDao(mysqlDaoArgs);
 class TopSecretInfoDao extends MysqlDao {
     get tableName() {return 'top_secret_info';}
 }
-const topSecretInfoDao = new TopSecretInfoDao(mysqlDaoArgs);
+const topSecretInfoDao = new TopSecretInfoDao(MysqlDaoArgsFactory.createFromJsObj({
+    mysqlService: mysqlService,
+    createVoFromDbRowFunction: function(dbRow) {
+        const args = {};
+        Object.keys(dbRow).forEach(columnName => {
```

Comment:
This looks like a dupe of createUserFromDbRow. Maybe make that function a generic function that is used by both anonymous classes.